backend/src/api/instances/index.ts

```diff
@@ -1,9 +1,21 @@
 import { Router } from 'opine/mod.ts';
 import { getHandler } from './get.handler.ts';
+import { postHandler } from './post.handler.ts';
 import instanceRouter from './[id]/index.ts';
-
+import { v4 } from 'std/uuid/mod.ts';
 const instancesRouter = Router();
+
+/**
+ * Validate the id parameter for all endpoints at /api/instances/[id]/*
+ */
+instancesRouter.param('id', (req, _res, next, id) => {
+  if (!v4.validate(id)) return next('route');
+  req.params['id'] = String('' + id);
```

Comment:
```suggestion
  req.params['id'] = id.toString();
```
Works for strings and integers